src/cargo/util/toml_mut/manifest.rs

```diff
@@ -363,6 +363,30 @@ impl LocalManifest {
         Ok(())
     }
 
+    /// Add feature entry to a Cargo.toml.
+    pub fn insert_into_feature_table(
+        &mut self,
+        table_path: &[String],
```

Comment:
The name of this function says what table we are editing, so we shouldn't need to accept `table_path`

Reply:
thanks and solved.